modules/installation-special-config-storage.adoc

```diff
@@ -46,72 +84,82 @@ Mirroring is available only for user-provisioned infrastructure deployments on {
 ====
 
 [id="installation-special-config-storage-procedure_{context}"]
-== Configuring disk encryption or mirroring
+== Configuring disk encryption and mirroring
 
-.Procedure
+You can enable and configure encryption and mirroring during an {product-title} installation.
 
-To enable boot disk encryption or mirroring during {product-title} deployment:
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
+* You installed Butane on your installation node.
++
+[NOTE]
+====
+Butane is a command-line utility that {product-title} uses to provide convenient, short-hand syntax for writing machine configs, as well as for performing additional validation of machine configs. For more information, see the _Creating machine configs with Butane_ section.
+====
++
+* You have access to a {op-system-base-full} 8 system that can be used to generate a thumbprint of the Tang exchange key.
+
+.Procedure
 
 . If you want to use TPM v2 to encrypt your cluster, check to see if TPM v2 encryption needs to be enabled in the BIOS on each node. This is required on most Dell systems. Check the manual for your computer.
 
 . If you want to use Tang to encrypt your cluster, follow these preparatory steps:
 
-.. Access a Red Hat Enterprise Linux server from which you can configure the encryption settings.
-
-.. Set up or access an existing Tang server. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions. See link:https://youtu.be/2uLKvB8Z5D0[Securing Automated Decryption New Cryptography and Techniques] for a presentation on Tang.
+.. Set up a Tang server or access an existing one. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#network-bound-disk-encryption_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Network-bound disk encryption] for instructions.
 
-.. Install the `clevis` package, if it is not already installed:
+.. Install the `clevis` package on a {op-system-base} 8 node, if it is not already installed:
```

Comment:
"node" could imply a cluster node, which is not actually necessary here.  Any {op-system-base} machine with network access to the Tang server is sufficient.

Reply:
I have changed "node" to "machine" for each of the RHEL 8 references in the module.

Can you please expand on the second sentence in your comment? Does this mean that any earlier release of RHEL prior to 8 is also valid? This would also have an impact on the note which states that "{op-system-base} 8 provides Clevis version 15...". Thanks.

Reply:
The second sentence was just meant to emphasize the first.  I think it's fine to stick with RHEL 8 in these instructions.

Reply:
Ok great, thank you for confirming that.